_includes/footer.html

```diff
@@ -1,17 +1,35 @@
 <footer class="footer-section">
   <div class="footer-container">
-    <div class="footer-logo">
-      <img
-        src="../assets/images/bg/Pyconf 2020-03.svg"
-        alt="Pyconf hyderabad"
-      />
+    <div class="conf-image">
+      <img src="../2020/assets/images/bg/Pyconf-2020-03.svg" alt="Pyconf hyderabad"/>
     </div>
-    <div class="footer-links">
-      <h3>Links</h3>
-      <a href="dummy">dummy</a>
-      <a href="dummy">dummy</a>
-      <a href="dummy">dummy</a>
-      <a href="dummy">dummy</a>
+
+    <div class="copyright-details">
+      <div class="copyright-text">Copyright 2020 Pyconf hyderabad</div>
+      <div class="copyright-logo"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png"></div>
     </div>
+
+    <div class="social-media-details">
+
+      <div>
+        <a href="https://twitter.com/pyconfhyd" rel="nofollow noreferrer" target="_blank">
+          <img src="../2020/assets/images/logo/twitter-logo.svg" alt="twitter">
+        </a>
+      </div>
+
+      <div>
+        <a href="https://github.com/HydPy/pyconfhyd2020" rel="nofollow noreferrer" target="_blank">
+          <img src="../2020/assets/images/logo/github-logo.svg" alt="github">
+        </a>
+      </div>
+
+      <div>
+        <a href="https://hydpy.zulipchat.com/" rel="nofollow noreferrer" target="_blank">
+          <img src="../2020/assets/images/logo/chat-logo.svg" alt="zulip">
```

Comment:
these url references might not work if you are on speaker description page, please verify.
Click on speaker image on homepage to navigate to speaker description page.

Reply:
speaker page does not have footer ?

Reply:
It does have footer.